tests/plugins/test_plugins_manager.py

```diff
@@ -163,14 +170,16 @@ class AirflowAdminMenuLinksPlugin(AirflowPlugin):
 
             appbuilder_menu_items = [mock.MagicMock()]
 
-        with mock_plugin_manager(plugins=[AirflowAdminViewsPlugin(), AirflowAdminMenuLinksPlugin()]):
+        with mock_plugin_manager(
+            plugins=[AirflowAdminViewsPlugin(), AirflowAdminMenuLinksPlugin()]
+        ), caplog.at_level(logging.WARNING):
             from airflow import plugins_manager
 
-            # assert not logs
-            with self.assertRaises(AssertionError), self.assertLogs(plugins_manager.log):
```

Comment:
I found this a huge code smell, and a funny/hard to follow way of saying "does not log".

So I switched it all out to pytests `caplog` fixture.